style.css

```diff
@@ -0,0 +1,141 @@
+body {
+  overflow: hidden;
+  font-family: 'Roboto', sans-serif;
+}
+
+.template5-main-mobile {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 24px;
+  background: #191765;
+  width: 375px;
+  height: 5415px;
+}
+
+.app-bar {
+  position: absolute;
+  width: 420px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  line-height: 16px;
+  padding-bottom: 20px;
+}
+
+.time {
+  position: absolute;
+  width: 50px;
+  height: 16px;
+  left: 5px;
+  top: 16px;
+}
+
+.name {
+  position: static;
+  padding-top: 50px;
+  padding-left: 10px;
+  color: #edecff;
+}
+
+.top-circle {
+  position: absolute;
+  height: 563px;
+  width: 513px;
+  left: -106px;
+  top: 0;
+  border-radius: 0;
+}
+
+.ellipse1 {
+  position: absolute;
+  height: 160px;
+  width: 160px;
+  right: 321px;
+  left: -106px;
+  top: 97px;
+  bottom: 5158px;
+  border-radius: 0;
+  background: #edecff;
+  filter: blur(50px);
+}
+
+.ellipse2 {
+  position: absolute;
+  height: 68px;
+  width: 68px;
+  left: 339px;
+  top: 495px;
+  border-radius: 0;
+  bottom: 5158px;
+  background: #edecff;
+  filter: blur(50px);
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  position: static;
+  height: 194px;
+  width: 360px;
+  left: 24px;
+  right: 24px;
+  top: 40px;
+  border-radius: 0;
+}
```

Comment:
- [ ] You did an amazing job implementing design from [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) but it is having issue. Please fix it following the design.
Here you have a screenshot of how I am seeing the first section:
![image](https://user-images.githubusercontent.com/74506933/153674616-478773b5-86e6-4589-9cdd-3622569daafe.png)

Reply:
Thank you